maintainer/travis/build_docker.sh

```diff
@@ -17,15 +17,16 @@ if [ -z "$image" ]; then
 fi
 
 if [ "$TRAVIS_OS_NAME" = "linux" ]; then
-	image=espressomd/buildenv-espresso-$image
-	docker run -u espresso --env-file $ENV_FILE -v ${PWD}:/travis -it $image /bin/bash -c "git clone /travis && cd travis && maintainer/travis/build_cmake.sh"
+	image=espressomd/buildenv-espresso-$image:python
```

Comment:
If the tag is not hardcoded here, users cant check on travis in advance. Is this what we want?